### ISSUE TITLE
NTR: Muscle biopsy: phosphoglycerate mutase low (Lab: pathology: biopsies & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9082]
New term request Muscle biopsy: phosphoglycerate mutase low (segal_050203185009): PGAM2  UMLS: 5393166

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000197
- label: Reduced muscle phosphoglycerate mutase activity
- parents: [Abnormal enzyme activity in muscle tissue (HP:0034526)](https://hpo.jax.org/app/browse/term/HP:0034526)
- pmid: 
- def: Activity of the enzyme Phosphoglycerate mutase (PGM) in muscle tissue is below the lower limit of normal. PGM is an enzyme that catalyzes step 8 of glycolysis (EC 5.4.2.11).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9082
- synonyms:
